contracts/adventurer/src/adventurer_meta.cairo

```diff
@@ -9,6 +9,7 @@ struct AdventurerMetadata {
     rank_at_death: u8, // 2 bits in storage
     delay_stat_reveal: bool, // 1 bit in storage
     golden_token_id: u8, // 8 bits in storage
+    launch_tournament_winner_token_id: u32, // 32 bits in storage
```

Comment:
@starknetdev note addition of launch_tournament_winner_token_id: u32 to AdventurerMetadata